workers/unity/Packages/com.improbable.gdk.core/Worker/Worker.cs

```diff
@@ -9,19 +9,49 @@
 
 namespace Improbable.Gdk.Core
 {
+    /// <summary>
+    ///     Represents a SpatialOS worker.
+    /// </summary>
     public class Worker : IDisposable
     {
+        /// <summary>
+        ///     The origin of the worker in local Unity space.
+        /// </summary>
         public readonly Vector3 Origin;
+
+        /// <summary>
+        ///     The type of the worker.
+        /// </summary>
         public readonly string WorkerType;
+
+        /// <summary>
+        ///     The worker ID.
+        /// </summary>
         public readonly string WorkerId;
 
+        /// <summary>
+        ///     THe logger for this worker.
+        /// </summary>
         public ILogDispatcher LogDispatcher;
 
+        /// <summary>
+        ///     The connection to the SpatialOS runtime.
+        /// </summary>
+        /// <remarks>
+        ///    May be null.
```

Comment:
when is it null?

Reply:
good to copy the above comment verbatim (including trollface) into remarks

Reply:
or, `<banter></banter>` section

Reply:
If only C# XML documentation was down with the kids